s2cloudless/utils.py

```diff
@@ -0,0 +1,61 @@
+"""
+Module with utilities
+"""
+
+S2_BANDS = ['B01', 'B02', 'B03', 'B04', 'B05', 'B06', 'B07', 'B08', 'B8A', 'B09', 'B10', 'B11', 'B12']
+MODEL_BAND_IDS = [0, 1, 3, 4, 7, 8, 9, 10, 11, 12]
+
+DATA_EVALSCRIPT_TEMPLATE = """
+//VERSION=3
+function setup() {{
+  return {{
+    input: [{{
+      bands: [{bands}],
+      units: "{input_units}"
+    }}],
+    output: {{
+      bands: {band_number},
+      sampleType: "{output_sample_type}"
+    }}
+  }};
+}}
+{metadata_evalscript}
+function evaluatePixel(sample) {{
+  return [{sample_bands}];
+}}
+"""
+
+METADATA_EVALSCRIPT_TEMPLATE = """
+function updateOutputMetadata(scenes, inputMetadata, outputMetadata) {
+  outputMetadata.userData = {
+    "norm_factor":  inputMetadata.normalizationFactor
+  }
+}
+"""
+
+
+def get_s2_evalscript(all_bands=True, reflectance=True):
```

Comment:
Just a convenience in case someone would like to prototype and want to avoid dealing with normalization factors. This is for example in the example notebook.

I did however decide to change the default values of this function to `all_bands=False, reflectance=False`.

Reply:
Ok.